.github/workflows/ci-app-pr-environment-update.yml

```diff
@@ -0,0 +1,12 @@
+name: CI App PR Environment Update
+on:
+  workflow_dispatch:
+  # !! Uncomment the following lines once you've set up the dev environment and are ready to enable PR environments
+  # pull_request:
```

Comment:
you didn't want to add the following commented out? `types: [opened, synchronize, reopened]`

Reply:
Those are the default behavior, so without them specified that's what Github does.

Reply:
Would be curious on why a comment should not be included that mentioned that though?

Reply:
Oh I see what you mean. I don't think it makes sense to do something like 

```
# pull_request:
#   types: [opened, synchronize, reopened]
```

because it's confusing why we would have commented out code.

I think it could be helpful to have a comment like

```
# By default, the pull_request event triggers on event types: "opened", "synchronize", and "reopened"
# pull_request:
```